test-data/simple.json

```diff
@@ -25,6 +25,54 @@
       "name": "Feature.C",
       "enabled": true,
       "strategies": []
+    },
```

Comment:
We don't have any client spec tests for custom strats, it's convenient to have these for tests